client/src/app.jsx

```diff
@@ -5,7 +5,7 @@ import ReactDOM from 'react-dom';
 // eslint-disable-next-line no-unused-vars
 import $ from 'jquery';
 import Overview from './components/overview/overview.jsx';
-import Main from './components/relatedProducts/Main.jsx';
+import RelatedProducts from './components/relatedProducts/RelatedProducts.jsx';
```

Comment:
Yay! Thank you for changing the name here so that our naming is more consistent throughout all of our components